kratos/includes/exception.h

```diff
@@ -152,11 +152,11 @@ namespace Kratos
   ///@name Kratos Macros
   ///@{
 
-#define KRATOS_ERROR throw Exception("Error: ", KRATOS_CODE_LOCATION)
```

Comment:
sorry for the changes in the core
We will discuss this together with the macro issue